docs/source/common/lightning_module.rst

```diff
@@ -1070,23 +1074,41 @@ recurrent network trajectories."
 
     class MyModel(LightningModule):
 
-        def __init__(self):
+        def __init__(self, input_size, hidden_size, num_layers, output_size,
+                     truncated_bptt_steps, criterion):
             super().__init__()
+            self.lstm = nn.LSTM(
+                input_size=input_size,
+                hidden_size=hidden_size,
+                num_layers=num_layers,
+                batch_first=True,
+            )
+            self.linear = nn.Linear(hidden_size, outpur_size)
+            self.criterion = criterion
+
             # Important: This property activates truncated backpropagation through time
             # Setting this value to 2 splits the batch into sequences of size 2
-            self.truncated_bptt_steps = 2
+            self.truncated_bptt_steps = truncated_bptt_steps
```

Comment:
for illustration purposes, let's keep it explicit here.

Reply:
Sure, I will change that.